LICENSES/Apache-2.0.txt

```diff
@@ -200,9 +200,3 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
-
-
---------------------------------------------------------------------------------------------------------------------------
```

Comment:
Why is this shift necessary? Does the process require it? Because if not the LICENSE file is more common and I am not sure, that npm will understand the file in a subdirectory... (e. g. see license on the side: https://www.npmjs.com/package/@sap-cloud-sdk/core )

Reply:
Here is the reason why you have to remove these lines:
https://jam4.sapjam.com/wiki/show/JBQV5da04GwOp45levWo10
see 3->todos->a

Reply:
> e. g. see license on the side: https://www.npmjs.com/package/@sap-cloud-sdk/core

I guess the info comes from:
https://github.com/SAP/cloud-sdk/blob/master/package.json#L7
`package.json` also specifies the `homepage` and `repository`, which you can find just next to the license info.